Source/NSSwitch.m

```diff
@@ -0,0 +1,288 @@
+/* Implementation of class NSSwitch
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: Wed Apr  8 22:01:02 EDT 2020
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#import "AppKit/NSSwitch.h"
+#import "GNUstepGUI/GSTheme.h"
+
+@implementation NSSwitch
+
++ (void) initialize
```

Comment:
I just compared with NSScroller, another NSControl subclass without a cell and in some regards we are already better here. But some methods are missing that would normally go to the NSCell, for example stuff like acceptsFirstResponder. And there are the takeXXXValueFrom type of methods, that are also missing in NSScroller. Should we add these?

Reply:
I believe we don't need to.  The takeXXXValueFrom methods don't seem to exist on NSSwitch.  Also, some methods on this don't seem to have any effect (such as setAcceptsFirstResponder:) per tests.

Reply:
The test I wrote for this doesn't work... investigating.

Reply:
Just look through the code of NSScroller yourself and decide what we could use here.

Reply:
I reran it... it appears to work.  Also, in Xcode the takeXXX methods do not appear in the connection inspector (as they do for NSButton and others).

Reply:
We might have to add a few more methods later on, when people start to use this class.